webpack.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = (env, options) => {
     return {
         mode: options.mode,
         devtool:
-            options.mode !== 'production' ? 'inline-source-map' : undefined,
+            options.mode !== 'production' ? 'eval-source-map' : undefined,
```

Comment:
In Django 4.0, [ManifestStaticFilesStorage](https://docs.djangoproject.com/en/5.0/ref/contrib/staticfiles/#django.contrib.staticfiles.storage.ManifestStaticFilesStorage) checks for missing files, and the build output with `inline-source-map` was failing because of a link to a missing `bootstrap.esm.js.map`. Changing the [devtool option](https://webpack.js.org/configuration/devtool/) to `eval-source-map`, which is their _Recommended choice for development builds with high quality SourceMaps_, fixed the issue